json/src/main/scala/io/sphere/json/generic/package.fmpp.scala

```diff
@@ -133,11 +133,16 @@ package object generic extends Logging {
   ): ToJSON[T] = {
     val jsonClass = getJSONClass(classTag[T].runtimeClass)
     val _fields = jsonClass.fields
-    val _withTypeHint = jsonClass.typeHint.isDefined
+    val hintField: JField = {
+      if (jsonClass.typeHint.isDefined) {
+        val th = jsonClass.typeHint.get
+        JField(th.field, JString(th.value))
+      } else null
+    }
```

Comment:
what about a pattern matching to avoid option.get ?
```suggestion
    val hintField: JField = jsonClass.typeHint match {
      case Some(th) => JField(th.field, JString(th.value))
      case None => null  
    }
```

Reply:
Sure, I had the low level brain activated and forgot about nice things.